crates/bevy_core_pipeline/src/taa/taa.wgsl

```diff
@@ -0,0 +1,190 @@
+// References:
+// https://www.elopezr.com/temporal-aa-and-the-quest-for-the-holy-trail
+// http://behindthepixels.io/assets/files/TemporalAA.pdf
+// http://leiy.cc/publications/TAA/TAA_EG2020_Talk.pdf
+// https://advances.realtimerendering.com/s2014/index.html#_HIGH-QUALITY_TEMPORAL_SUPERSAMPLING
+
+const HISTORY_BLEND_RATE: f32 = 0.1;
+const MIN_HISTORY_BLEND_RATE: f32 = 0.015;
```

Comment:
From where do these come? And what do they mean?

Reply:
I made it up, and decided on these values through trial and error. I'll add some docs.

Reply:
I did see mention of 90% / 10% somewhere, but the MIN_HISTORY_BLEND_RATE value I don't know.